server/src/lib/db/mysql.ts

```diff
@@ -15,7 +15,7 @@ type MysqlOptions = {
 const DEFAULTS = {
   user: 'voiceweb',
   database: 'voiceweb',
-  password: '',
+  password: null,
```

Comment:
Do we still need to change this? Is `null` equivalent to empty string for mysql?